packages/@sanity/schema/src/types/object.js

```diff
@@ -1,8 +1,9 @@
 import {pick, keyBy, startCase} from 'lodash'
 import {lazyGetter} from './utils'
 import createPreviewGetter from '../preview/createPreviewGetter'
+import guessSortConfig from '../sort/guessSortConfig'
 
-const OVERRIDABLE_FIELDS = ['jsonType', 'type', 'name', 'title', 'description', 'options', 'inputComponent']
+const OVERRIDABLE_FIELDS = ['jsonType', 'sorting', 'type', 'name', 'title', 'description', 'options', 'inputComponent']
```

Comment:
Sorry for derailing, but should I have added `hidden` to this list, too?

Reply:
Yeah! Looks like `readOnly` is missing there too.